drivers/hmis/app/models/hmis/workflow_execution/engine.rb

```diff
@@ -12,6 +12,10 @@
 # task assignments, and message processing according to the workflow template.
 module Hmis::WorkflowExecution
   class Engine
+    # Constants for referral event types
+    ACCEPT_REFERRAL = 'accept_referral'
+    REJECT_REFERRAL = 'reject_referral'
```

Comment:
This breaks the current delineation we have where the code in `drivers/hmis/app/models/hmis/workflow_execution` and `../workflow_definition` is _not referral specific_. The referral-related stuff should remain in the `ce/` module. So I'd suggest moving these to the referral message handler, and maybe naming them something more descriptive like ACCEPT_REFERRAL_MESSAGE and REJECT_REFERRAL_MESSAGE.

Reply:
Ok, thank you, sorry